docs/server_api/index.rst

```diff
@@ -2,12 +2,8 @@
 Bodhi Server APIs
 =================
 
-fedmsg API
-==========
-
-The fedmsgs bodhi sends
-`are documented by the Fedora fedmsg project <https://fedora-fedmsg.readthedocs.io/en/latest/topics.html#bodhi>`_.
-
+.. TODO: add Fedora Messaging documentation when we can generate it from the schemas.
+   See https://github.com/fedora-infra/fedora-messaging/issues/120
```

Comment:
Let's not put TODOs in the code. I actually do have a personal task in task warrior to remind me to document our schemas. We could also file an issue about it if you like.